docs/development.md

```diff
@@ -50,6 +50,7 @@ This repository contains the main code, but two more repositories are included d
 
 - To build the application, follow the Theia IDE [prerequisites](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites).
 - This project recommends using [Visual Studio Code (VS Code)](https://code.visualstudio.com/) for the development.
+- Optionally, you might need to build the [Arduino CLI](https://github.com/arduino/arduino-cli), the [Arduino Language Server](https://github.com/arduino/arduino-language-server), and other tools from the sources. For more details, refer to the Arduino CLI's [prerequisites section](https://arduino.github.io/arduino-cli/latest/CONTRIBUTING/#prerequisites).
```

Comment:
```suggestion
- The build system might also need to build the [Arduino CLI](https://github.com/arduino/arduino-cli), the [Arduino Language Server](https://github.com/arduino/arduino-language-server), and other tools from the sources. In this case it is also necessary to have the build prerequisites for those projects installed. For more details, refer to the Arduino CLI's [prerequisites section](https://arduino.github.io/arduino-cli/latest/CONTRIBUTING/#prerequisites).
```

- Remove "optionally", because this will give the reader the impression that they are free to skip it if they like. Although in theory it is "optional" for someone who is willing and able to change the version specified in `package.json` to a release version, it is not optional for someone who only wants to build with the currently configured tool versions, in cases where a non-release version happens to be configured.
- Replace the "you might need to build" wording. This will give the reader the impression that they must directly build the tools, when actually the build system will do it for them automagically as long as the prerequisites are available.
- Explicitly state that this is about installing the prerequisites of the tools